env/env.go

```diff
@@ -14,4 +14,5 @@ const (
 	InjectorMountSysrqTrigger = "CHAOS_INJECTOR_MOUNT_SYSRQ_TRIGGER"
 	InjectorTargetPodHostIP   = "TARGET_POD_HOST_IP"
 	InjectorChaosPodIP        = "CHAOS_POD_IP"
+	MyPodName                 = "MY_POD_NAME"
```

Comment:
```suggestion
	ControllerPodName                 = "CONTROLLER_POD_NAME"
```
If that's ok for you, because I added almost the same variable for node name and it is named `CONTROLLER_NODE_NAME`, so we keep the same format?

Reply:
But this definitely isn't the Controller Pod Name though?

Reply:
Changed to Injector